### ISSUE TITLE
JumpUrl v8.0 incompatibility with TYPO3_9 version

### ISSUE THREAD

**Issue by jokumer** [jokumer/direct_mail#21]
Within develop of TYPO3 CMS v9.5 compatible version of direct_mail v6.0,
the required extension friendsoftypo3/jumpurl is compatible with v7.8 only. Update to v8.0 breaks for links.
`#1359987599 Exception
The calculated Jump URL hash ("juHash") did not match the submitted "juHash" query parameter.`

**Comment by AnatoliZich** [comment]
I don't think you need to adapt to jumpurl 8.x IMHO. Just leave the dependency to 7.8 in your composer.json.

For TYPO3 v10 however the story is completely different. My TYPO3 v10 fork migrates the jumpurl handling to middlewares as needed with jumpurl 8.x
